logistics/scripts/run-tests.ps1

```diff
@@ -5,7 +5,7 @@
 
 & "$PSScriptRoot\modules\load-path-resolver.ps1"
 
-$testAssemblies = (Get-ChildItem -recurse -File $((Get-RepositoryRoot "ed-fi-ods") + "\*Tests.dll") | Where-Object { $_.FullName -match "\\bin\\?" -and $_.FullName -notmatch "\\net48\\?" -and $_.fullName -notmatch "ApprovalTests.dll" })
+$solutionPath = (Get-RepositoryResolvedPath "Application/Ed-Fi-Ods.sln")
```

Comment:
Does this ticket really call for running all of the tests, both unit and integration? Or have all of the integration tests now been removed?

Reply:
I believe this was already running all the tests. We had added this filter during our conversion to dotnet core to keep it from running some of the older tests that were being run by the NUnit runner.

Reply:
OK. Typically I setup scripts to run unit and integration tests separately, so that the fast unit tests will run and give you feedback before running integration tests. That could be a separate improvement request if needed in the future. No change requested right now.